src/apps/dashboard/features/logs/components/LogItemList.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FunctionComponent } from 'react';
+import type { LogFile } from '@jellyfin/sdk/lib/generated-client/models/log-file';
+import { List, ListItem, ListItemButton, ListItemText } from '@mui/material';
+import { useApi } from 'hooks/useApi';
+import datetime from 'scripts/datetime';
+
+type LogItemProps = {
+    logs: LogFile[];
+};
+
+const LogItemList: FunctionComponent<LogItemProps> = ({ logs }: LogItemProps) => {
+    const { api } = useApi();
+
+    const getLogFileUrl = (logFile: LogFile) => {
```

Comment:
Let's add a `TODO` comment to replace this with `getUri` in the SDK once it is available.